libdnf.spec

```diff
@@ -35,6 +35,7 @@ BuildRequires:  gcc
 BuildRequires:  libsolv-devel >= %{libsolv_version}
 BuildRequires:  pkgconfig(librepo)
 BuildRequires:  pkgconfig(check)
+BuildRequires:  redhat-rpm-config
```

Comment:
Why did you file it at GNOME? I told you to file it in Red Hat Bugzilla (note the "rhbz" in my message!).